frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/ai/ai_writer_block_component.dart

```diff
@@ -634,3 +670,25 @@ class MainContentArea extends StatelessWidget {
     );
   }
 }
+
+bool isDocumentEmptyForAI(Document document) {
+  if (document.root.children.isEmpty) {
+    return true;
+  }
+
+  final markdown = documentToMarkdown(
```

Comment:
It would be better to add a comment here to explain why we use the `customDocumentToMarkdown` function.

Reply:
OK!